src/vmware/CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 ## 2.0.0 (TBD)
```

Comment:
Do we replace TBD with a specific date before merging this PR?

Reply:
There are more features coming. 2.0 is not ready just yet. This PR is. Would you prefer something else there?

Reply:
No problem. Do you want me to merge it?

Reply:
Please do. I do not have access.